backend-go/internal/server/server.go

```
@@ -31,9 +36,28 @@ func New(e *echo.Echo, conf *config.Config) (*CodePair, error) {
 		return nil, fmt.Errorf("failed to dial mongo: %w", err)
 	}
 
+	var storageClient storage.Client
+	if conf.Storage.Provider == "s3" {
+		storageClient, err = s3.NewClient(conf.Storage.S3.Bucket)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create s3 client: %w", err)
+		}
+	} else if conf.Storage.Provider == "minio" {
+		storageClient, err = minio.NewClient(
+			conf.Storage.Minio.Bucket, 
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)